pkg/services/policer/policer.go

```diff
@@ -12,15 +12,22 @@ import (
 	"github.com/nspcc-dev/neofs-node/pkg/services/replicator"
 	"github.com/nspcc-dev/neofs-node/pkg/util/logger"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
+	"github.com/panjf2000/ants/v2"
 	"go.uber.org/zap"
 )
 
+// NodeLoader provides functions to fetch application load statistic.
+type nodeLoader interface {
```

Comment:
```suggestion
type NodeLoader interface {
```

Reply:
Should it be public? I am not sure it needs to be exported somewhere out of policer package.